scripts/cleanCache.command

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+BASE_DIR=~/Library/ApplicationSupport/Mattermost
```

Comment:
Should this be a variable to support Windows/Linux? Or do we not need this script to run automatically

Reply:
I added this one by mistake, I want to create this as a tool for people in case they run into problems (mostly devs) while doing upgrades/downgrades. 
and yes, it would need to be changed depending on OS and available tools.

Also, I want it to be kind of 'click and forget' and that's basically the reason why I didn't add it before. I'll remove it